ompi/mca/bcol/basesmuma/bcol_basesmuma_buf_mgmt.c

```diff
@@ -99,6 +99,9 @@ int bcol_basesmuma_free_buff( sm_buffer_mgmt * buff_block,
      *   associated with this bank have been freed.
      */
     assert(generation == buff_block->ctl_buffs_mgmt[memory_bank].bank_gen_counter);
+#if !OPAL_ENABLE_DEBUG
```

Comment:
Yeah, that PR came in recently with the weird `(void)` technique -- I didn't put 2 and 2 together back then to realize that it was because of a variable that only exists in debug builds.